engines/ultima/detection_tables.h

```diff
@@ -119,203 +150,58 @@ static const UltimaGameDescription GAME_DESCRIPTIONS[] = {
 
 #ifdef ENABLE_ULTIMA6
 	// GOG Ultima VI
-	{
-		{
-			"ultima6",
-			0,
-			AD_ENTRY1s("converse.a", "5065716423ef1389e3f7b4946d815c26", 162615),
-			Common::EN_ANY,
-			Common::kPlatformDOS,
-			ADGF_NO_FLAGS,
-			GUI_OPTIONS_ULTIMA6
-		},
-		GAME_ULTIMA6,
-		0
-	},
-
-	// GOG Ultima VI - Enhanced
-	{
-		{
-			"ultima6_enh",
-			0,
-			AD_ENTRY1s("converse.a", "5065716423ef1389e3f7b4946d815c26", 162615),
-			Common::EN_ANY,
-			Common::kPlatformDOS,
-			ADGF_NO_FLAGS,
-			GUI_OPTIONS_ULTIMA6
-		},
-		GAME_ULTIMA6,
-		GF_VGA_ENHANCED
-	},
+	ENTRY_ULTIMA6(AD_ENTRY1s("converse.a", "5065716423ef1389e3f7b4946d815c26", 162615),
```

Comment:
No, please do not do nested macros like this. Just add these three arguments as parameters as we do in other engines